scripts/figma-import/icons.js

```diff
@@ -0,0 +1,78 @@
+const config = require('../../figma-import.config')
```

Comment:
I'd suggest to keep this config in the same directory as the script to not pollute the top-level directory